fp.sql

```diff
@@ -20,6 +20,33 @@ WITH gather_survey_data AS (
     GROUP BY
         site_short
 ),
+get_at_data AS (
+    SELECT
+        site_short AS at_site,
+        --% of college students saying that they know how to apply for Emergency Fund in PAT advising rubric
+        SUM(
+            CASE
+                WHEN e_fund_c = 'EF_G' THEN 1
+                ELSE 0
+            END
+        ) AS indicator_efund,
+    FROM
+        `data-warehouse-289815.salesforce_clean.contact_at_template`
+    WHERE
+        college_track_status_c = '15A'
+        AND(
+            (
+                CURRENT_DATE() < '2021-07-01'
```

Comment:
@jvolan123 Think I know what is happening here with the date filters, but would be good to confirm with comment

Reply:
@jvolan123 Also want to review the fp_12_fafsa_complete_num field. Apologies as I didn't notice this before

Reply:
re: comment, will do. 

re: fp_12_fafsa_complete_num. 
@isabelreyna What is your question / concern?

Reply:
@jvolan123  Mostly wondering which KPI this is for. Is it for EFC?

Reply:
@isabelreyna , got it. just added in a comment in my sql. 

--% of high school seniors with EFC by end of March. Right now we are not accounting for the timebound part as we don't currently have a way to track when it was completed.

Reply:
chatted on slack. Changed the field we use to calc numerator. Looking at the actual EFC value field instead of the EFC source field. Notes here for reference